File: zendrive-ui/src/api/models/auth/dto/EditUserDto.ts
export class EditUserDto {
	id: string;
	roles: string[];
	enabled: boolean;
	locked: boolean;

	constructor(id: string, roles: string[], enabled: boolean, locked: boolean) {
		this.id = id;
		this.roles = roles;
		this.enabled = enabled;
		this.locked = locked;
	}
}

File: zendrive-api/src/main/java/com/zendrive/api/rest/models/dto/admin/EditUserDto.java
package com.zendrive.api.rest.models.dto.admin;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class EditUserDto {
	private Long id;
	private List<String> roles;
	private boolean enabled;
	private boolean locked;
}
